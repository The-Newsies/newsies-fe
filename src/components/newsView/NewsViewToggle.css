.link {
  text-decoration: none;
  color: black;
  margin-right: 5px;
}

.viewToggle {
  display: flex;
  left: 0;
}