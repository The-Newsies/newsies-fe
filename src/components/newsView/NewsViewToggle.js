import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import styles from './NewsViewToggle.css';

export default function NewsViewToggle({ handleTrending }) {
  return (
    <span>
      <NavLink to='/trending' onClick={handleTrending} className={styles.link} activeStyle={{ 
        fontWeight: 'bold', 
        color: 'white', 
        background: 'red', 
      }}>Trending</NavLink>
      <NavLink to='/myNews' className={styles.link} activeStyle={{ 
        fontWeight: 'bold', 
        color: 'white', 
        background: 'red', 
      }}>My News</NavLink>
    </span>
  );
}

NewsViewToggle.propTypes = {
  handleTrending: PropTypes.func.isRequired
};
