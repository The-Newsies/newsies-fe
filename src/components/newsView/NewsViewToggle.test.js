import React from 'react';
import { shallow } from 'enzyme';
import NewsViewToggle from './NewsViewToggle';

describe('NewsViewToggle component', () => {
  it('renders NewsViewToggle', () => {
    const wrapper = shallow(<NewsViewToggle handleMyNewsView={() => { }} handleTrendingView={() => { }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
