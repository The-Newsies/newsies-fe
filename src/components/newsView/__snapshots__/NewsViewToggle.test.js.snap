// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsViewToggle component renders NewsViewToggle 1`] = `
<span>
  <a
    onClick={[Function]}
  >
    Trending
  </a>
  <a
    onClick={[Function]}
  >
    My News
  </a>
</span>
`;
